openshift/pipelines/Jenkinsfile.test-to-prod

```diff
@@ -127,10 +127,15 @@ pipeline {
     }
 
     stage("Database migration") {
-      options { timeout(time: 5, unit: "MINUTES") }
+      options { timeout(time: 15, unit: "MINUTES") }
+      environment {
+        DB_PASSWORD = sh(script: "oc -n jcxjin-prod get secret pims-db-prod-secret -o jsonpath='{.data.DB_PASSWORD}' | base64 -d", returnStdout: true).trim()
+        DB_CONNECTION_STRING = sh(script: "oc -n jcxjin-prod get cm pims-api-prod-db-client -o jsonpath='{.data.CONNECTION_STRINGS_PIMS}'", returnStdout: true).trim()
+      }
```

Comment:
This "extracts" OpenShift secrets into environment variables required by the migrations script below